styles.css

```diff
@@ -0,0 +1,111 @@
+:root {
+  --primary-color: #3f37c9;
+  --secondary-color: #e5e7eb;
+  --white: #f3f4f6;
+  --blue: #0000ff;
+  --logo-font: "Haas Grot Text R Web", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+a {
+  text-decoration: none;
+}
+.wrapper {
+  width: 85%;
+  margin: auto;
+  height: 100vh;
+}
+.navbar {
+  padding: 2.5rem 1rem;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.nav-right {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+}
+.nav-right-list-items {
+  margin: 0.1rem 1rem;
```

Comment:
For very small units you can use px

Reply:
Thanks for pointing that out, will do the changes!